contracts/staking/tests/ValidatorManagerTests.t.sol

```diff
@@ -143,7 +143,7 @@ abstract contract ValidatorManagerTest is Test {
         // TODO: implement
     }
 
-    function testResendEndValidation() public {
+    function testResendEndValidation() public virtual {
```

Comment:
Made virtual so it can be overridden in the PoS test file, in order to induce uptime logic.